Cargo.toml

```diff
@@ -38,3 +38,7 @@ serde_json = "1.0"
 thiserror = "1.0"
 tiny-keccak = "1.5"
 web3 = "0.8"
+
+[dev-dependencies]
+anyhow = "1.0"
+rustversion = "1.0"
```

Comment:
Shouldn't this only be used by the example feature and thus be specified in the feature section?

Reply:
`example` feature is for generating example contract code to get included in the documentation. I think I might rename the feature so it doesn't get confused with `cargo run --example`

Reply:
Created issue #27

Reply:
My question was more since rustversion is only used by the example feature, can we make this dependency be only downloaded and built when compiled with example feature enabled.

Reply:
Its used regardless of `example` feature as our `async` example gets built with or without that feature.